vms/avm/service.go

```diff
@@ -157,19 +160,24 @@ func (service *Service) GetUTXOs(r *http.Request, args *GetUTXOsArgs, reply *Get
 	}
 
 	chainID := ids.ID{}
+	if args.SourceChain == "" {
```

Comment:
It seems like we set `chainID` and then don't do anything with it

Reply:
chainID is used to check if it matches the chainID of this vm as well as getting passed in to getAtomicUTXOs

Reply:
:facepalm: